### PR TITLE
Webhook and Alternate Email Endpoints

### PR DESCRIPTION
**Pull request by cameronbowie**

### CODE REVIEW

File: lib/users/index.js [hunk 1]
```diff
@@ -33,13 +34,21 @@ exports.create = function(options) {
...
-    getUser                : listAllUsers,
-    listAllUsers           : listAllUsers,
-    getCurrentUser         : getCurrentUser,
-    addUser                : addUser,
-    addUserAndSendEmail    : addUserAndSendEmail,
-    updateUser             : updateUser,
-    removeUser             : removeUser
+  var buildUserObject = function () {
+    var userObject = {
+      getUser                : listAllUsers,
```
Comment (armstnp) [inline lib/users/index.js:39]: This is an odd one.  Is there an API function for getting a user by ID?

File: lib/webhooks/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    return utils.get(_.extend(optionsToSend, getOptions), callback);
+  };
+
+  var resetSharedSecret = function(postOptions, callback) {
+    optionsToSend.url = buildUrl(postOptions.webhookId) + '/resetsharedsecret';
+    return utils.post(_.extend(optionsToSend, postOptions), callback);
+  };
+
+  var buildUrl = function(webhookId) {
+    return options.apiUrls.webhooks + (webhookId || '');
```
Comment (armstnp) [inline lib/webhooks/index.js]: Does this need a slash before the ID?

File: lib/users/index.js (hunk 1 above)
Comment (cameronbowie) [reply lib/users/index.js:39]: There are two endpoints that are the same with the exception of the trailing id. If the id is specified, Get User is called. If it is not specified, Get All Users is called

File: lib/webhooks/index.js (hunk 2 above)
Comment (cameronbowie) [inline lib/webhooks/index.js:40]: No, all of the API URL constants have trailing slashes

File: lib/users/index.js (hunk 1 above)
Comment (armstnp) [reply lib/users/index.js:39]: Gotcha.  Mostly looks weird, but probably harmless.

**Pull request merged**